Frontend/src/api/mocks/get_competition_info.ts

```diff
@@ -0,0 +1,39 @@
+import getCompetitionInfo from '../competition/get/get_competition_info'
+import { CompetitionInfo } from '../types'
+import {
+  CLOSED_COMPETITION,
+  COMMENT_REQUIRED,
+  FAVOURITES_COMPETITION,
+  NOT_YET_OPEN,
+  OPEN_COMPETITION,
+  OPEN_WITH_PAYMENTS,
+} from './fixtures'
+
+export default async function getCompetitionInfoMock(
+  competitionId: string
+): Promise<CompetitionInfo> {
+  switch (competitionId) {
+    case 'KoelnerKubing2023': {
+      return OPEN_COMPETITION
+    }
+    case 'RheinNeckarAutumn2023': {
+      return OPEN_WITH_PAYMENTS
+    }
+    case 'HessenOpen2023': {
+      return CLOSED_COMPETITION
+    }
+    case 'ManchesterSpring2024': {
+      return NOT_YET_OPEN
+    }
+    case 'FMCFrance2023': {
+      return COMMENT_REQUIRED
+    }
+    case 'PickeringFavouritesAutumn2023': {
+      return FAVOURITES_COMPETITION
+    }
+    default: {
+      // This allows non mocked response when debugging a certain competition
```

Comment:
The function name is a bit misleading then.

Reply:
I would like to keep that functionality though, so we can more easily handle bug reports. Do you have any good idea for a name?

Reply:
Maybe add `...WithRealFallback` on to the end, or something to that effect?

Reply:
renamed